### ISSUE TITLE
Bought the app today, the iPhone app works but not on Apple Watch (9)

### ISSUE THREAD

**Issue by jbuggie** [itsMeDavidV/Watchla-Support#314]
Set up the app per instructions and the iPhone app works. The Tesla got a virtual washla key added. However, the Watchla app on the watch does not have any info about the Tesla.  Tried "Watch Key Setup" on the watch but it just sit there at "initializing"

**Comment by rogersmj** [comment]
Yeah it’s been in a state kind of like this for about two weeks since I bought it. The developer has posted a couple minor updates and asked me to try those but the watch app still doesn’t work. No word in a couple of days.

**Comment by Kennethol** [comment]
Have you update the app ind the watch

**Comment by jbuggie** [comment]
multiple times still not working

**Comment by itsMeDavidV** [comment]
We're currently finishing up an overhaul on how Watchla manages its authentication among shared extensions (eg: complications, watch app, shortcuts) in response to Tesla's new and very strict authentication protocol. This should solve this issue. This solution will be available in beta very soon and then on the App Store shortly after if all goes well. If anyone would like to test this in beta before it goes out to the AppStore, please reach out and mention this issue: https://watchla.app/contact
